src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body * {
+  box-sizing: border-box;
+  font-family: "Avenir", arial, sans-serif;
+  font-weight: 300;
+}
+
+.searchbar {
+  position: relative;
+  padding-left: 62px;
+  margin: 12px 0 20px;
+  height: 70px;
+}
+
+.searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus {
+  outline: none;
+  background: linear-gradient(180deg, rgb(255, 255, 255)
+  0%, hsl(240, 6%, 97%) 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.small {
+  position: relative;
+  height: 42px;
+  width: 100%;
+  padding-left: 50px;
+}
+
+.small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small:focus {
+  outline: none;
+  background: linear-gradient(180deg, rgb(255, 255, 255)
+  0%, hsl(240, 6%, 97%) 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+#big::placeholder {
```

Comment:
checklist p.1 Get used to style all elements using classes. And don't increase selectors specificity unless completely necessary